modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/block/processing/BlockNotAcceptedReason.scala

```diff
@@ -25,6 +25,9 @@ case class ValidationFailed(reasons: NonEmptyList[BlockValidationError]) extends
 @derive(eqv, show)
 case class ParentNotFound(parent: BlockReference) extends BlockRejectionReason
 
+@derive(eqv, show)
+case object SnapshotOrdinalUnavailable extends BlockRejectionReason
```

Comment:
for future, I think that blocks should not be rejected because of the fact the node is not initialized yet.
either we should return "not ready yet" on the endpoint not to receive data for consensus (in a similar way we do for snapshots endpoint) or we should queue txs/blocks and start processing them when the node is fully initialized. what do you think? can you create a ticket for that?
cc: @ryle-goehausen @kpudlik @TheMMaciek

Reply:
Should it extend `BlockAwaitReason` instead, or are you suggesting something else altogether?

Reply:
I'm not familiar with the errors within the block processing, would have to check precisely. I think that creating the ticket to offload this for future would be best